Recrutify/Recrutify.Services/Validators/CreateCandidateValidator.cs

```diff
@@ -35,14 +35,19 @@ public CreateCandidateValidator()
             RuleForEach(c => c.Contacts)
                 .NotNull()
                 .NotEmpty();
-            RuleFor(c => c.Location)
+            RuleFor(c => c.Location.City)
+                .NotNull()
+                .NotEmpty();
+            RuleFor(c => c.Location.Country)
```

Comment:
don't you want validate length of city and country names

Reply:
done